pylint_odoo/checkers/itp_checks.py

```diff
@@ -59,7 +59,7 @@ def _is_string_instance(obj):
         settings.DESC_DFLT
     ),
     'E%d91' % settings.BASE_OMODULE_ID: (
-        'Module\'s image is absent or it\'s not in images/ folder. Check attribute "images" in module\'s __manifest__.py',
+        'Module\'s image is absent or it\'s not in images/ folder or image has invalid name. Check attribute "images" in module\'s __manifest__.py',
```

Comment:
:-1:

Reply:
if will be invalid image's name than pylint speak about this too

Reply:
The only invalid name is absent file